src/data_provider/testtool/Readme.md

```diff
@@ -0,0 +1,32 @@
+# Data Provider Testing Tool
+## Index
+1. [Purpose](#purpose)
+2. [Compile Wazuh](#compile-wazuh)
+3. [How to use the tool](#how-to-use-the-tool)
+
+## Purpose
+The Data Provider Testing Tool was created to test and validate the data obtained by the module's execution. This tool works as a black box where an user will be able execute it and analyze the output data as desired.
+
+## Compile Wazuh
+In order compile the solution on a specific wazuh target, the project needs to be built either in release or debug/test mode.
+```
+make TARGET=server|agent <DEBUG=1 TEST=1>
```

Comment:
Remove TEST option from this file, the test tool not are compiled in this mode.

Reply:
Done in last commit.